frontend/src/components/TournamentManager.js

```diff
@@ -322,68 +325,48 @@ const TournamentManager = () => {
 
         <Show when={selectedTournamentID() > 0 && selectedTournament()}>
           <div class="relative overflow-x-auto my-5">
-            <table class="w-full text-sm text-left text-gray-500 dark:text-gray-400">
-              <thead class="text-xs text-gray-700 uppercase bg-gray-50 dark:bg-gray-700 dark:text-gray-400">
-                <tr>
-                  <th scope="col" class="px-6 py-3">
-                    Seeding
-                  </th>
-                  <th scope="col" class="px-6 py-3">
-                    Team Name
-                  </th>
-                  <th scope="col" class="px-6 py-3">
-                    Team ID
-                  </th>
-                </tr>
-              </thead>
-              <tbody>
-                <For each={Object.keys(tournamentSeeding())}>
-                  {seed => (
-                    <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
-                      <th
-                        scope="row"
-                        class="px-6 py-4 font-medium text-gray-900 whitespace-nowrap dark:text-white"
-                      >
-                        {seed}
-                      </th>
-                      <td class="px-6 py-4">
-                        {teamsMap()[tournamentSeeding()[seed]]}
-                      </td>
-                      <td class="px-6 py-4">{tournamentSeeding()[seed]}</td>
-                    </tr>
+            <div class="text-blue-500 text-xl font-bold mb-2">Seeding</div>
+            <div class="flex flex-row mb-5 w-1/2">
```

Comment:
`w-1/2` should only be active on large screens. On mobile screens, the full width can be used.

Reply:
Ah yes, would some media queries solve this ?

Reply:
You can add a prefix like `md` or `sm` to specify the size at which we want to start using a different rule. Something like would work, I think. 

```
<div class="w-1/2 md:w-full">
  <!-- ... -->
</div>
```

Reply:
yeah, i just added this. Thanks @punchagan